draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -225,7 +227,13 @@ when, and only when, they appear in all capitals, as shown here.
 
 This document additionally uses the TLS presentation language defined in {{Section 3 of !RFC8446}}.
 
-# Overview
+## Time {#time}
+
+All time computations in this document are represented by POSIX timestamps, defined in this document to be integers containing a number of seconds since the Epoch, defined in Section 4.16 of {{POSIX}}. That is, the number of seconds after 1970-01-01 00:00:00 UTC, excluding leap seconds. A UTC time is converted to a POSIX timestamp as described in {{POSIX}}.
+
+Durations of time are integers, representing a number of seconds not including leap seconds. They can be added to POSIX timestamps to produce other POSIX timestamps.
+
+The current time is a POSIX timestamp determined by converting the current UTC time to seconds since the Epoch. One POSIX timestamp is said to be before (respectively, after) another POSIX timestamp if it is less than (respectively, greater than) the other value.
```

Comment:
Should you move this down with the other definitions (issuance time? etc)?

Reply:
That's actually the parameters to a CA. The terminology bit is a little awkward... right now, the terminology section just defines enough terms to talk about the overview and the rest is inline. It's... probably not quite the right model. I didn't want to drop a wall of terms without how they go together, but everything inline was differently awkward so... now we have a bit of both. :-/

Thinking I'll leave it as-is for now, since I imagine the organization will need to change more drastically as the document evolves anyway.